res/values/strings.xml

```diff
@@ -827,22 +826,22 @@
 
 
     <!-- iOS permissions, copy from "deltachat-ios/Info.plist", which is used on missing translations in "deltachat-ios/LANG.lproj/InfoPlist.strings" -->
-    <string name="InfoPlist_NSCameraUsageDescription">Delta Chat uses your camera to take and send photos and videos and to scan QR codes.</string>
-    <string name="InfoPlist_NSContactsUsageDescription">Delta Chat uses your contacts to show a list of email addresses you can write to. Delta Chat has no server, your contacts are not sent anywhere.</string>
-    <string name="InfoPlist_NSMicrophoneUsageDescription">Delta Chat uses your microphone to record and send voice messages and videos with sound.</string>
-    <string name="InfoPlist_NSPhotoLibraryUsageDescription">Delta Chat will let you choose which photos from your library to send.</string>
-    <string name="InfoPlist_NSPhotoLibraryAddUsageDescription">Delta Chat wants to save images to your photo library.</string>
+    <string name="InfoPlist_NSCameraUsageDescription">Your camera scans QR codes for logins, and captures photos and videos to send directly.</string>
+    <string name="InfoPlist_NSContactsUsageDescription">A list of e-mail addresses in your contacts is assembled. It is not sent anywhere.</string>
+    <string name="InfoPlist_NSMicrophoneUsageDescription">Your mic is used to record messages and videos with sound to send directly.</string>
+    <string name="InfoPlist_NSPhotoLibraryUsageDescription">Access to your library lets you pick existing files, images and videos to send.</string>
+    <string name="InfoPlist_NSPhotoLibraryAddUsageDescription">Access to your gallery is needed to save images and video you capture.</string>
 
 
     <!-- android specific strings, developers: please take care to remove strings that are no longer used! -->
     <string name="pref_background_notifications">Background notifications</string>
-    <string name="pref_background_notifications_explain">Uses a background connection to your server and requires ignored battery optimizations.</string>
-    <string name="pref_background_notifications_rationale">To maintain connection to your email server and receive messages in the background, ignore battery optimizations in the next step.\n\nDelta Chat uses few resources and takes care not to drain your battery.</string>
+    <string name="pref_background_notifications_explain">Uses a background connection to your mail server and requires ignored battery optimizations.</string>
```

Comment:
It seems to me that there is a "consistency" problem here.  Until now we have used e-mail and here mail.

Reply:
There was a problem, because no one convention was used, I think that may be what leads to so many erroneous variants being used. It should always be "mail server", unfortunately.

Reply:
Conventions should probably be followed, if this does not create confusion for the end user. 🤔